visualization/app/codeCharta/ui/codeMap/codeMap.label.service.ts

```diff
@@ -169,16 +168,21 @@ export class CodeMapLabelService {
 		const { scaling } = this.storeService.getState().appSettings
 		const { margin } = this.storeService.getState().dynamicSettings
 
-		const multiplier = scaling.clone().divide(this.previousScaling)
+		const labelHeightDifference = new Vector3(0, this.LABEL_HEIGHT_COEFFICIENT * margin * this.LABEL_SCALE_FACTOR, 0)
 
 		for (const label of this.labels) {
-			const labelHeightDifference = new Vector3(0, this.LABEL_HEIGHT_COEFFICIENT * margin * this.LABEL_SCALE_FACTOR, 0)
-			label.sprite.position.sub(labelHeightDifference).multiply(multiplier).add(labelHeightDifference)
+			const multiplier = scaling.clone()
```

Comment:
Please see also line 49. Why do we have two times `const multiplier = scaling.clone()`?